.github/issue_template.md

```diff
@@ -0,0 +1,20 @@
+### Steps to reproduce
+
+(First please check that this issue is not already solved as [described
+here](https://github.com/feathersjs/feathers/blob/master/.github/contributing.md#report-a-bug))
+
+- [ ] Tell us how you broke things. The more detailed the better.
+- [ ] If you can, please create a simple example that reproduces the issue and link to a gist, jsbin, repo, etc.
+
+### Expected behavior
+Tell us what should happen
+
+### Actual behavior
+Tell us what happens instead
+
+### System configuration
+**Module versions** (especially the part that's not working):
+
+**NodeJS version**:
+
+**Operating System**:
```

Comment:
Should we ask for the client configuration here, too? Browser version and module loader (if applicable)